README.md

```diff
@@ -200,7 +200,8 @@ Adds new sound file to X Sounds database.
 
 **Request** \
 Requires content type to be `multipart/form-data` with fields:
-- `soundName`: name of the sound to be added (like _"bruh"_)
+- `soundName`: name of the sound to be added (like _"bruh"_
+- `tags` (optional): tag list separated with comma (like _"jp2, jan pawel, jean paul"_)
```

Comment:
1. Let's keep the readme context free
2. "comma separated list" would sound better

So I suggest something like this:

```suggestion
- `tags` (optional): comma separated list of tags (like _"tag1,tag2,tag3"_)
```

Same thing in the line above - instead of `bruh`, more generic name would be better like `cool sound` or something. You're also missing closing bracket on that line.

Reply:
Fixed